test/genesisProtocol.js

```diff
@@ -151,8 +151,8 @@ describe("GenesisProtocol", () => {
       avatar: dao.avatar.address,
     });
     assert.isOk(result);
-    assert.equal(result.thresholdConstA, 1);
-    assert.equal(result.thresholdConstB, 1);
+    assert.equal(web3.fromWei(result.thresholdConstA).toNumber(), 2);
```

Comment:
from where 2 and 10 come from ?
Would it will be easier to import this default values from somewhere..as when someday it will be changed this code can be relax?

Reply:
Yes, see https://github.com/daostack/arc.js/issues/171

Will probably create a json file for these values.

Reply:
why not including that on this pr ?

Reply:
>why not including that on this pr ?

Trying to keep tickets simpler, isolate the work, and so I can focus on the task.  Ideally I would be able to import code, not just json, but that is going to take focus and time.

Reply:
it suppose to be part of this pr.
as this pr change GP params. why double work ?

Reply:
the "double work" is trivial.  #171 explains why the alternative is not so trivial (particularly when importing code).